.github/workflows/superset-frontend.yml

```diff
@@ -121,24 +121,6 @@ jobs:
           files: merged-output/coverage-summary.json
           slug: apache/superset
 
-  core-cover:
```

Comment:
NOTE: this forced 100% test coverage across `core/`. Dropped it as we're bringing more things in here. Could be nice to keep the 100% requirement on parts of it, but gets in the way of velocity. Test now covered as part of the main repo tests. We had double test coverage here before which is a bit wasteful.